### ISSUE TITLE
Pokemon list does not include Unown forms

### ISSUE THREAD

**Issue by SimplyBLGDev** [PokeAPI/pokeapi#591]
The list of pokemon served at https://pokeapi.co/api/v2/pokemon/?limit=2000 does not include any of the Unown forms for some reason.

As a result of this the encounter table can't reference any of the Unown forms, I have not found any way to fix this or what is causing it.

**Comment by Kronopt** [comment]
It includes the unown pokemon, not form: `{"name":"unown","url":"https://pokeapi.co/api/v2/pokemon/201/"}`.
If you follow the url you can find the forms there:

```
"forms":[
	{
		"name": "unown-a",
		"url": "https://pokeapi.co/api/v2/pokemon-form/201/"
	},
	{
		"name": "unown-b",
		"url": "https://pokeapi.co/api/v2/pokemon-form/10001/"
	},
	{
		"name": "unown-c",
		"url": "https://pokeapi.co/api/v2/pokemon-form/10002/"
	},
	{
		"name": "unown-d",
		"url": "https://pokeapi.co/api/v2/pokemon-form/10003/"
	},
	{
		"name": "unown-e",
		"url": "https://pokeapi.co/api/v2/pokemon-form/10004/"
	},
	{
		"name": "unown-f",
		"url": "https://pokeapi.co/api/v2/pokemon-form/10005/"
	}
]
```

**Comment by SimplyBLGDev** [comment]
Yes, but the problem still stands, it can't be referenced from the encounters table and some encounters require the specific forms, the other forms are on that list for that reason.

**Comment by SimplyBLGDev** [comment]
I'm adding the missing encounter from Gen 3, I need to edit the existing Unown encounters in the Tanoby Chambers so that they properly point to the specific forms of Unown in each chamber, I need the Unown forms to be added to the Pokemon List or information on how can I add them myself.

**Comment by Kronopt** [comment]
I don't think they should be added as individually to the "pokemon" endpoint as they are just one pokemon (with a single pokedex entry) with multiple forms.

Can you explain how you're implementing that new encounter?

**Comment by SimplyBLGDev** [comment]
I need to add the counters to the tanoby ruins, different unowns are assigned to different slots in there, but the encounters table references the Pokemon table, which doesn't have the unown forms necessary to do this; Other Pokemon forms with a single pokedex entry have separate entries in the Pokemon endpoint, I don't see why unown should be any different.

**Comment by Kronopt** [comment]
I don't think I'm the best person to discuss this.
@Naramsim?

**Comment by SimplyBLGDev** [comment]
I'll also be trying to contribute these corrections to the upstream repo Veekun/pokedex (when I'm done with at least the first 4 gens since their PR approval process is way longer), I'll try to do that before the next wave of encounters arrive so that there's no conflict between the projects in the future so I'm trying to get everything in before that happens (it doesn't seem to be anytime soon since there's no PR for that, they're focusing on the stats changes and that kind of stuff)

**Comment by C-Garza** [comment]
Unown is not the only pokemon that has forms without a unique pokemon id. If you look at the `pokemon_id` column in the [pokemon_forms.csv](https://github.com/PokeAPI/pokeapi/blob/master/data/v2/csv/pokemon_forms.csv) you can see there quite a few, like Arceus and Vivillon's forms.

**Comment by Kronopt** [comment]
I wonder why that's the case.
Would unique unown `pokemon_id`s solve your issue, @SimplyBLGDev?

**Comment by SimplyBLGDev** [comment]
Yes, unique pokemon_id s would be necessary to get the encounter right.

**Comment by SimplyBLGDev** [comment]
Hey, I've been noticing there have been multiple recent issues related to these particular forms that are not stored in the same way as others, another problem I've noticed is that pokemon-form s that have no pokemon entry don't have sprites for every generation, because of this unown A has it's sprites and icons from every generation but the unown variants only have gen 3 sprites (for some reason) while other pokemon with multiple forms have every sprite, for example deoxys has multiple pokemon entries, one at https://pokeapi.co/api/v2/pokemon/386/ that only mentions it's normal form and *doesn't* have any of the forms listed under the 'forms' field, it's other forms are in the 10,000's like here https://pokeapi.co/api/v2/pokemon/10001/ that has it's attack form and all of it's information, this has lead to some very strange and inconsistent names in the sprites sub project, like here:
![image](https://user-images.githubusercontent.com/61010688/112921819-d3c6e200-90e1-11eb-9820-eef37bbb5fa3.png)

**Comment by SimplyBLGDev** [comment]
Just to clarify the info is missing from pokeAPI not the sprites project, altough the sprites project is missing sprites on certain *games* such as crystal's unown forms even though gold and silver have them.

**Comment by justingolden21** [comment]
It seems like unown should be listed as forms. Open to other opinions here. I believe they're like furfrou trims or flabebe colors etc. Different skins that count as different pokemon but also function the same other than aesthetic (of course forms can function differently as well).
